nebula-exchange/src/test/scala/com/vesoft/nebula/exchange/NebulaGraphMock.scala

```diff
@@ -0,0 +1,138 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package scala.com.vesoft.nebula.exchange
+
+import com.vesoft.nebula.client.graph.NebulaPoolConfig
+import com.vesoft.nebula.client.graph.data.HostAddress
+import com.vesoft.nebula.client.graph.net.NebulaPool
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ListBuffer
+
+class NebulaGraphMock {
+
+  @transient val nebulaPoolConfig = new NebulaPoolConfig
+  @transient val pool: NebulaPool = new NebulaPool
+  val address                     = new ListBuffer[HostAddress]()
+  address.append(new HostAddress("127.0.0.1", 9669))
+
+  val randAddr = scala.util.Random.shuffle(address)
+  pool.init(randAddr.asJava, nebulaPoolConfig)
+
+  def mockStringIdGraph(): Unit = {
+    val session = pool.getSession("root", "nebula", true)
+
+    val createSpace = "CREATE SPACE IF NOT EXISTS testSST1(partition_num=10);" +
+      "USE testSST1;" + "CREATE TAG IF NOT EXISTS person(col1 string, col2 fixed_string(8), col3 int8, col4 int16, col5 int32, col6 int64, col7 date, col8 datetime, col9 timestamp, col10 bool, col11 double, col12 float, col13 time);" +
+      "CREATE EDGE IF NOT EXISTS friend(col1 string, col2 fixed_string(8), col3 int8, col4 int16, col5 int32, col6 int64, col7 date, col8 datetime, col9 timestamp, col10 bool, col11 double, col12 float, col13 time);";
+    val createResp = session.execute(createSpace)
+    if (!createResp.isSucceeded) {
+      close()
+      sys.exit(-1)
+    }
+
+    Thread.sleep(3000)
+    val insertTag =
```

Comment:
why don't loading from an extra resource file? such as json file ?

Reply:
The purpose is to mock Nebula data to support the tests, I don't think it's necessary to load ngql from extra resource file.